src/bokeh/models/widgets/tables.py

```diff
@@ -858,6 +858,37 @@ def __init__(self, *args, **kwargs) -> None:
     The height of each row in pixels.
     """)
 
+    @staticmethod
+    def to_data_table(data, columns=None, formatters={}, **kwargs) -> DataTable:
+        """ Create a simple table from a pandas dataframe, dictionary or ColumnDataSource.
+
+        Parameters:
+        data (pandas DataFrame or dict or ColumnDataSource): The data to create the table from.
+        columns (list, optional): A list of column names to use. If None, use all columns.
+        formatters (dict, optional): A formatter to be applied to all columns or specified columns.
+
+        Returns:
+        bokeh.models.widgets.tables.DataTable: The created DataTable.
```

Comment:
indent, also returns usually just specify the type unless there is a real need to elaborate
```suggestion
            DataTable
```